examples/jre/build.gradle

```diff
@@ -12,7 +12,11 @@ repositories {
 }
 
 dependencies {
-    implementation 'com.fiskaly.sdk:fiskaly-sdk:1.2.200-jre'
+    //implementation 'com.fiskaly.sdk:fiskaly-sdk:1.2.200-jre'
```

Comment:
I'm not certain whether this is the best way to make it use the local version of the SDK (I get the impression that it's possible to use a local Maven repository to do it instead, without changing the dependencies in the gradle file, but I wasn't able to get that working due to inexperience with Maven in general), so I left this line commented out in case it was better to go back to it. If you think that the way I've done it is okay, I will remove the commented-out line.